SPECS/kernel/config

```diff
@@ -6513,6 +6515,7 @@ CONFIG_IIO_ST_ACCEL_I2C_3AXIS=m
 # CONFIG_AD7291 is not set
 # CONFIG_AD7606_IFACE_PARALLEL is not set
 # CONFIG_AD799X is not set
+# CONFIG_ADI_AXI_ADC is not set
```

Comment:
result of
https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/b70042e4dc7fdfbe28d2f518f56aa643d1256b41
https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/d3513f1261a0615eea6cf6839a57abedc648cc4c